dev-vcs/git-remote-hg/git-remote-hg-1.0.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+DESCRIPTION="Semi-official Mercurial bridge from Git project"
+HOMEPAGE="https://github.com/mnauw/git-remote-hg"
+SRC_URI="https://github.com/mnauw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="
+	app-text/asciidoc
+"
+DEPEND="
+	dev-lang/python:2.7
```

Comment:
No. You should use python-single-r1 for this. And make sure to install the correct shebangs.